android-o/PictureInPicture/PictureInPicture/Properties/AndroidManifest.xml

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android" android:versionCode="1" android:versionName="1.0" package="com.xamarin.PictureInPicture">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.xamarin.PictureInPicture" android:versionCode="1" android:versionName="1.0">
 	<uses-sdk android:targetSdkVersion="26" android:minSdkVersion="26" />
-	<application android:allowBackup="true" android:icon="@mipmap/icon" android:label="@string/app_name" android:theme="@style/AppTheme"></application>
+	<application android:allowBackup="true" android:label="@string/app_name" android:icon="@mipmap/icon" android:theme="@style/AppTheme"></application>
 </manifest>
```

Comment:
The delta in this file is just extra noise in the PR, we should not include these changes.